articles/azure/azs-how-create-spn-cli.md

```diff
@@ -48,7 +48,10 @@ To log in and manage your resources via SPN you'll need to create an Azure appli
 
 - [Service Principal Name commands for 2018-03-01-hybrid profile](https://docs.microsoft.com/en-us/cli/azure/ad/sp?view=azure-cli-2018-03-01-hybrid)
 
-- [Create an Azure Service Principal Name with Azure CLI 2.0 for 2018-03-01-hybrid profile](azs-how-create-spn-cli.md)
+    > [!NOTE]
+    > There is currently no document for the 2019-03-01-hybrid profile.
+
+- [Create an Azure Service Principal Name with Azure CLI 2.0 for 2019-03-01-hybrid profile](azs-how-create-spn-cli.md)
 
 ## Overview of the creation process for Azure Stack Hub SPN
```

Comment:
In this section, a `SPN Password` variable is provided but isn't used in the code?

Reply:
Fixed in commit 4d5ac62